airbyte-webapp-e2e-tests/cypress/pages/newConnection.ts

```diff
@@ -0,0 +1,25 @@
+type ConnectorType = "source" | "destination";
```

Comment:
File should be called `newConnectionPage`

Reply:
Renamed